plugin/trino-mongodb/src/test/java/io/trino/plugin/mongodb/BaseMongoConnectorSmokeTest.java

```diff
@@ -24,6 +24,7 @@ protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
     {
         switch (connectorBehavior) {
             case SUPPORTS_CREATE_SCHEMA:
+            case SUPPORTS_RENAME_TABLE:
```

Comment:
create schema and rename table are unrelated, so i'd somewhat prefer to un-lump them into separate options

Reply:
Reverted